src/plugins/Makefile.am

```diff
@@ -280,9 +280,9 @@ if PLUGIN_PROCDUMP2
 sources += procdump2/procdump2.cpp
 sources += procdump2/procdump2.h
 sources += procdump2/private2.h
-sources += procdump2/writer.cpp
-sources += procdump2/writer.h
-sources += procdump2/minidump2.h
+sources += procdump/writer.cpp
+sources += procdump/writer.h
+sources += procdump/minidump.h
```

Comment:
This doesn't look right. One plugin should never include files/headers from another one. In this case it's pretty clear one would only be using one version of the plugin so whatever cppcheck is complaining about is likely due to it trying with both being enabled. Adding an assert to ensure that never happens would be the right solution.

Reply:
I fix it using namespaces